staging/src/k8s.io/apiserver/pkg/util/flowcontrol/apf_controller.go

```diff
@@ -444,6 +444,19 @@ func (cfgCtlr *configController) digestConfigObjects(newPLs []*flowcontrol.Prior
 	return suggestedDelay, utilerrors.NewAggregate(errs)
 }
 
+func (cfgCtlr *configController) generatePatchBytes(condition flowcontrol.FlowSchemaCondition) ([]byte, error) {
```

Comment:
nit: does it need to be a `configController` receiver function?

Reply:
that is a good question; I did this because this function was called only within the object.

Reply:
I think either choice would be acceptable.  Since this function does not use the configController, I think it would make more sense to be a function rather than a method of configController.

As a function, it should have a more specific name.  Perhaps something like `makeFlowSchemaConditionPatch`.